apps/block_scout_web/lib/block_scout_web/views/layout_view.ex

```diff
@@ -195,13 +195,103 @@ defmodule BlockScoutWeb.LayoutView do
 
   def other_explorers do
     if Application.get_env(:block_scout_web, :link_to_other_explorers) do
-      decode_other_explorers_json(Application.get_env(:block_scout_web, :other_explorers, []))
+      decode_json(Application.get_env(:block_scout_web, :other_explorers, []))
     else
       []
     end
   end
 
-  defp decode_other_explorers_json(data) do
+  def bridges_list do
```

Comment:
can we refactor these methods to use the same implementation? like

```elixir
def get_list(list_key) do
    if Application.get_env(:block_scout_web, list_key) do
      try do
        :block_scout_web
        |> Application.get_env(list_key)
        |> Parser.parse!(%{keys: :atoms!})
      rescue
        _ ->
          []
      end
    else
      []
    end
  end

def bridges_list, do: get_list(:bridges)
#etc
```

Reply:
Yes, definitely